SPECS/rook/rook.spec

```diff
@@ -0,0 +1,1192 @@
+%package rookflex
+Summary:        Rook FlexVolume driver
+Group:          System/Filesystems
+
+%description rookflex
+Rook uses FlexVolume to integrate with Kubernetes for performing storage
+operations.
+
+################################################################################
+# Rook and Ceph manifests metadata
+################################################################################
+%package k8s-yaml
+Summary:        Kubernetes YAML file manifests for deploying a Ceph cluster
+Group:          System/Management
+BuildRequires:  ceph
+BuildArch:      noarch
+
+%description k8s-yaml
+This package contains examples of yaml files required to deploy and run the
+Rook-Ceph operator and Ceph clusters in a Kubernetes cluster.
+
+################################################################################
+# Rook ceph operator helm charts
+################################################################################
+%package ceph-helm-charts
+Summary:        Rook Ceph operator helm charts
+Group:          System/Management
+BuildArch:      noarch
+
+%description ceph-helm-charts
+Helm helps manage Kubernetes applications. Helm Charts define,
+install, and upgrade Kubernetes applications. Rook is a
+cloud-native storage orchestrator for Kubernetes, providing
+the platform, framework, and support for a diverse set of storage
+solutions to integrate with cloud-native environments.
+
+This package contains Helm Charts for Rook.
+
+################################################################################
+# Rook integration test binary metadata
+################################################################################
+%package integration
+Summary:        Application which runs Rook integration tests
+Group:          System/Benchmark
+
+%description integration
+This package is intended to be used only for testing. Please don't install it
+in production environments.
+
+Rook's integration tests conveniently get built into a standalone binary. The
+tests require a running Kubernetes cluster, and the image being tested must be
+pushed to all Kubernetes cluster nodes as 'rook/ceph:master'. They also require
+that 'kubectl' works without additional connection arguments from the system
+which will run the binary. The integration tests can be flaky and are best run
+on a Kubernetes cluster which has not previously run the integration tests.
+
+The list of possible integration test suites can be gotten from the integration
+binary with the argument [-test.list '.*']. A subset of test suites can be run
+by specifying a regular expression (or a specific test suite name) as an
+argument to [-test.run]. All Ceph test suites can be run with the argument
+[-test.run 'TestCeph'].
+
+################################################################################
+# Build section
+################################################################################
+%define _buildshell /bin/bash
+Vendor:         Microsoft Corporation
```

Comment:
Please move these to the top of the spec, similar to what's happening in other ones. I think running a linter through the spec should also do that for you.